nucypher/characters/control/specifications/fields/file.py

```diff
@@ -0,0 +1,33 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import os
+from marshmallow import fields
+
+from nucypher.characters.control.specifications.exceptions import InvalidInputData, InvalidNativeDataTypes
+from nucypher.characters.control.specifications.fields.base import BaseField
+
+
+class FileField(BaseField, fields.String):
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        with open(value, 'rb') as plaintext_file:
+            plaintext = plaintext_file.read()  # TODO: #2106 Handle large files
```

Comment:
So we deserialize a file by reading it from disk? I'm not sure which contexts it will be used in, but it just seems a little... error-prone? Even if it's the same machine, what if the file is missing?

Reply:
For some context read the `specifications` package, This is part of the EnricoEncrypt specification used in the CLI, and in web controllers, and is mapped from `--file`.  Types are validated and enforced prior to this step for missing or invalid files.